granian/_loops.py

```diff
@@ -34,12 +34,14 @@ def get(self, key: str) -> Callable[..., Any]:
 
 
 class BuilderRegistry(Registry):
-    __slots__ = []
+    __slots__ = ('_data',)
```

Comment:
`_data` slot is already inherited by the upper class. An empty list (and this is also why I used lists in place of tuples, makes the thing more readable IMHO) just avoid to add the local dictionary and should be fine.